showcase/src/app/app.component.scss

```diff
@@ -24,3 +24,20 @@
     }
   }
 }
+
+@media #{$mobile-only-query} {
+  .stark-app-sidebar {
+    .stark-app-sidenav-menu {
+      .stark-app-logo {
+        & i {
+          display: block;
+          width: 150px;
+          height: 60px;
+          margin-top: 0;
+          background-image: url(/assets/stark-core/logo/stark_logo_black.svg);
```

Comment:
Is this working? I don't think that this path is configured anywhere.... the right path should be the one you define in the `angular.json` file of the stark-core ;)

Reply:
It is working 😊